client/blocks/importer/wordpress/upgrade-plan/constants.tsx

```diff
@@ -49,23 +49,26 @@ export const UpgradePlanHostingTestimonials = [
 	},
 ];
 
-export const defaultHostingDetails = [
-	{
-		id: 'reduced-error-rate',
-		title: translate( 'Reduced error rate' ),
-		description: translate( '16% fewer errors' ),
-		icon: shield,
-	},
-	{
-		id: 'increased-speed',
-		title: translate( 'Increased speed' ),
-		description: translate( '30% faster' ),
+export const defaultHostingDetails: {
+	[ key: string ]: {
+		title: string;
+		description: string | ReactNode;
+		icon: JSX.Element;
+	};
+} = {
+	'higher-speed': {
+		title: translate( 'Higher speed' ),
+		description: translate( '30% faster.' ) + '*',
```

Comment:
See the base for this calc: p1716304442708379/1715972644.965929-slack-C02NWDZBL0H (78.33-47.95)